packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -579,34 +584,36 @@ const TagSubforumPage2 = ({classes}: {
         </div>
         <PostsListSortDropdown value={sortBy} options={subforumSortings} />
       </div>
-      {newDiscussionOpen && <div className={classes.newDiscussionContainer}>
-        {/* FIXME: bug here where the submit and cancel buttons don't do anything the first time you click on them, on desktop only */}
-        <CommentsNewForm
-          tag={tag}
-          tagCommentType={"SUBFORUM"}
-          successCallback={refetch}
-          type="reply" // required to make the Cancel button appear
-          enableGuidelines={false}
-          cancelCallback={() => setNewDiscussionOpen(false)}
-        />
-      </div>}
+      {newDiscussionOpen && (
+        <div className={classes.newDiscussionContainer}>
+          {/* FIXME: bug here where the submit and cancel buttons don't do anything the first time you click on them, on desktop only */}
+          <CommentsNewForm
+            tag={tag}
+            tagCommentType={"SUBFORUM"}
+            successCallback={refetch}
+            type="reply" // required to make the Cancel button appear
+            enableGuidelines={false}
+            cancelCallback={() => setNewDiscussionOpen(false)}
+          />
+        </div>
+      )}
       <MixedTypeFeed
-        firstPageSize={10}
+        firstPageSize={15}
```

Comment:
Replying to comments after the first page of results is still a bit glitchy so I've increased the initial page size to reduce the chance of this happening